falco/__version__.py

```diff
@@ -0,0 +1,8 @@
+__title__ = "falco"
+__description__ = "Python client and SDK for Falco."
+__url__ = "https://github.com/falcosecurity/client-py"
+__version__ = "0.1.0"
+__author__ = "The Falco Authors"
```

Comment:
__version__ should be dynamically generated, just a thought.

Reply:
I think it's overkill right now (also I don't know how to do that if we use semver)